src/test/optimizer/strategy/exists_reformulation_rule_test.cpp

```diff
@@ -136,7 +136,7 @@ TEST_F(ExistsReformulationRuleTest, ComplexSubquery) {
         node_table_a)));
 
   const auto expected_lqp =
-  JoinNode::make(JoinMode::Anti, equals_(node_table_a_col_a, node_table_b_col_a),
+  JoinNode::make(JoinMode::AntiRetainNulls, equals_(node_table_a_col_a, node_table_b_col_a),
```

Comment:
Are you sure that we need a `retain` and not a `discard` here?

Reply:
NOT EXISTS should be equivalent to JoinMode::AntiRetainNulls.